src/View/Helper/FormHelper.php

```diff
@@ -1046,6 +1049,10 @@ public function control(string $fieldName, array $options = []): string
             'templates' => [],
             'templateVars' => [],
             'labelOptions' => true,
+            'aria-label' => empty($options['placeholder']) ? $options['label'] : $options['placeholder'],
+            'aria-required' => $options['required'] == true ? 'true' : null,
+            'aria-invalid' => $this->isFieldError($fieldName) ? 'true' : null,
+            'aria-describedby' => $this->isFieldError($fieldName) ? $fieldName . '-aria' : null,
```

Comment:
Generally, do these ARIA attributes maybe need to _not_ apply to hidden inputs?

Reply:
I don't think they're required or useful for hidden inputs. Aria agents will skip hidden inputs by default.

Reply:
You're right, I didn't think of hidden fields. It probably shouldn't matter, but I'd rather do it so that ARIA isn't in the hidden fields.

Reply:
I modified the condition, now ARIA will be inserted into everything outside the hidden field and if it has a label or placeholder attribute.